src/servers/src/http/script.rs

```diff
@@ -96,6 +104,9 @@ pub async fn run_script(
     Query(params): Query<ScriptQuery>,
 ) -> Json<JsonResponse> {
     if let Some(script_handler) = &state.script_handler {
+        let catalog = params
+            .catalog
+            .unwrap_or_else(|| DEFAULT_CATALOG_NAME.to_string());
```

Comment:
Looks like throw an error is better. This function is entrypoint.

Reply:
But in our opensource project, we only have one catalog `greptime`, so i think using the default catalog is better than requiring user to pass this unnecessary parameter.